queries/c/indents.scm

```diff
@@ -1,28 +1,51 @@
 [
-  (compound_statement)
-  (preproc_arg)
+  (compound_statement) 
   (field_declaration_list)
   (case_statement)
   (enumerator_list)
-  (struct_specifier)
   (compound_literal_expression)
   (initializer_list)
-  (while_statement)
-  (for_statement)
-  (switch_statement)
+  (init_declarator)
```

Comment:
This will be the most controversial part anyways, so cc-ing @theHamsta 
Even with the new queries, all the tests still passed, so does this mean weak tests or over-coverage of the queries before?

Reply:
there were some problem with `compound_statement` only the last time I tried, but I would just judge based on a net-positive on the tests. When we discover new bugs with the new queries we can revert or find a better solution.